cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/ConsentStatusApiModel.kt

```diff
@@ -88,29 +104,52 @@ data class CcpaCS(
 
 @Serializable
 data class GdprCS(
-    @SerialName("applies") val applies: Boolean?,
-    @SerialName("gdprApplies") val gdprApplies: Boolean?,
-    @SerialName("categories") val categories: List<String>?,
-    @SerialName("consentAllRef") val consentAllRef: String?,
-    @SerialName("consentedToAll") val consentedToAll: Boolean?,
-    @SerialName("legIntCategories") val legIntCategories: List<String>?,
-    @SerialName("legIntVendors") val legIntVendors: List<String>?,
-    @SerialName("postPayload") val postPayload: PostPayload?,
-    @SerialName("rejectedAny") val rejectedAny: Boolean?,
-    @SerialName("specialFeatures") val specialFeatures: List<String>?,
-    @SerialName("vendors") val vendors: List<String>?,
-    @SerialName("addtlConsent") val addtlConsent: String?,
-    @SerialName("consentStatus") val consentStatus: ConsentStatus?,
-    @SerialName("cookieExpirationDays") val cookieExpirationDays: Int?,
-    @SerialName("customVendorsResponse") val customVendorsResponse: CustomVendorsResponse?,
-    @SerialName("dateCreated") val dateCreated: String?,
-    @SerialName("euconsent") val euconsent: String?,
-    @Serializable(with = GrantsSerializer::class) val grants: Map<String, GDPRPurposeGrants>?,
-    @Serializable(with = TcDataSerializer::class) val TCData: Map<String, JsonElement>?,
-    @SerialName("localDataCurrent") val localDataCurrent: Boolean?,
-    @SerialName("uuid") var uuid: String?,
-    @SerialName("vendorListId") val vendorListId: String?,
-    @SerialName("webConsentPayload") val webConsentPayload: JsonObject? = null,
+    @SerialName("applies")
+    val applies: Boolean?,
+    @SerialName("gdprApplies")
+    val gdprApplies: Boolean?,
+    @SerialName("categories")
+    val categories: List<String>?,
+    @SerialName("consentAllRef")
+    val consentAllRef: String?,
+    @SerialName("consentedToAll")
+    val consentedToAll: Boolean?,
+    @SerialName("legIntCategories")
+    val legIntCategories: List<String>?,
+    @SerialName("legIntVendors")
+    val legIntVendors: List<String>?,
+    @SerialName("postPayload")
+    val postPayload: PostPayload?,
+    @SerialName("rejectedAny")
+    val rejectedAny: Boolean?,
+    @SerialName("specialFeatures")
+    val specialFeatures: List<String>?,
+    @SerialName("vendors")
+    val vendors: List<String>?,
+    @SerialName("addtlConsent")
+    val addtlConsent: String?,
+    @SerialName("consentStatus")
+    val consentStatus: ConsentStatus?,
+    @SerialName("cookieExpirationDays")
+    val cookieExpirationDays: Int?,
+    @SerialName("customVendorsResponse")
+    val customVendorsResponse: CustomVendorsResponse?,
+    @SerialName("dateCreated")
+    val dateCreated: String?,
+    @SerialName("euconsent")
+    val euconsent: String?,
+    @Serializable(with = GrantsSerializer::class)
+    val grants: Map<String, GDPRPurposeGrants>?,
+    @Serializable(with = JsonMapSerializer::class)
+    val TCData: Map<String, JsonElement>?,
```

Comment:
Same for `grants` and `TCData`, missing `@SerialName`

Reply:
same here